foo_053_alerts_price_spikes/README.md

```diff
@@ -1,9 +1,12 @@
 ## Food Price Spikes Dataset Near Real-time Script
 This file describes the near real-time script that retrieves and processes the [Alert for Price Spikes (ALPS) dataset](https://documents.wfp.org/stellent/groups/public/documents/manual_guide_proced/wfp264186.pdf?_ga=2.155059965.418661181.1556120721-1045976685.1553722904) for [display on Resource Watch](https://resourcewatch.org/data/explore/foo053-Food-Price-Spikes).
 
-{Describe how the original data came from the source.}
+This dataset was created by processing two JSON files. One of them contained information about markets in each region and the other contained information about alert for price spikes (alps) for each commodities in each market. These two JSONs were processed to get alert for all food commodities in each market. Two separate tables were created using the processed data so that they could be uploaded to Carto. Along with the regular table, an additional 'interaction' table was created. This table was created because, at any given time, a market may be experiencing alert for price spikes for multiple commodities. The 'interaction' on the Resource Watch map is only able to display one row of data for a given geometry. In order to display information about multiple food commodities in one market, information from each commodities had to be combined into a single row of data. All information about each commodities were combined into a single field for each market, which is what is shown on the Resource Watch map. In order to transform the data from the JSONs to the data tables used by Resource Watch, the following changes were made:
```

Comment:
```suggestion
This dataset was created by processing two JSON files. One of them contained information about markets in each region and the other contained information about alerts for price spikes (ALPS). Two separate tables were created to store the data from the JSONs so that they could be uploaded to Carto. In order to transform the data from the JSONs to the data tables used by Resource Watch, the following changes were made:
```